scripts/qa_crowbarsetup.sh

```diff
@@ -2040,6 +2042,22 @@ function onadmin_post_allocate()
             ssh $node "getent passwd glance >/dev/null || \
                 useradd -r -g glance -u 450 -d /var/lib/glance -s /sbin/nologin -c \"OpenStack glance Daemon\" glance"
         done
+
+        if iscloudver 6plus && [[ $want_sbd = 1 ]] ; then
+            zypper --gpg-auto-import-keys -p http://download.opensuse.org/repositories/devel:/languages:/python/SLE_12_SP1/ --non-interactive install python-sh
+            wget --no-check-certificate https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/iscsictl.py
+            chmod +x iscsictl.py
+            ./iscsictl.py --service target --host $(hostname) --no-key
+
+            for clustername in data network services ; do
+                eval "cluster=\$clusternodes$clustername"
+                for node in $cluster ; do
+                    /iscsictl.py --service initiator --target_host $(hostname) --host $node --no-key
+                    SBD_DEV=$(ssh $node echo '/dev/disk/by-id/scsi-$(lsscsi -i | grep LIO | tr -s " " |cut -d " " -f7)')
+                    ssh $node "zypper --non-interactive install sbd; sbd -d $SBD_DEV create"
```

Comment:
Exactly: does not hurt, and easier that deciding to do it only for the first time.